posthog/utils.py

```diff
@@ -356,6 +357,7 @@ def render_template(
         "persisted_feature_flags": settings.PERSISTED_FEATURE_FLAGS,
         "anonymous": not request.user or not request.user.is_authenticated,
         "year_in_hog_url": year_in_hog_url,
+        "event_property_definitions": CORE_FILTER_DEFINITIONS_BY_GROUP.get("event_properties", {}),
```

Comment:
Doesn't look like we're using this, what was the idea here?

Reply:
Ah, I backed away from changing this all in go. This is how I was going to pass from the backend into the frontend so we could collapse the duplication... will tidy this

Reply:
```suggestion
```

Reply:
Oh yeah, that's kind of what I figured, but seemed vestigial at the same time. It makes major sense as a follow-up